gapir/client/connection.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/google/gapid/core/app/auth"
 	"github.com/google/gapid/core/log"
 	replaysrv "github.com/google/gapid/gapir/replay_service"
+	"github.com/google/gapid/gapis/service/severity"
```

Comment:
Using the original 'Severity' enum defined in 'gapis/service/service.proto' results into cyclic dependency. So I have to extract out the severity to its own package.